financial-connections/src/main/java/com/stripe/android/financialconnections/ui/components/TopAppBar.kt

```diff
@@ -198,8 +207,12 @@ internal val LazyListState.elevation: Dp
 private fun NavHostController.collectCanGoBackAsState(): State<Boolean> {
     val canGoBack = remember { mutableStateOf(false) }
```

Comment:
extremely nit: technically users can go back, should we maybe rename this to `shouldShowBackIcon` or similar?

Reply:
Yep, will rename it.